### PR TITLE
feat: Added Landing Page of E-com

### PR DESCRIPTION
**Pull request by xii-vi**
Added Landing Page of E-com

RE-Kick
.

### CODE REVIEW

**Review by haarsh24 — APPROVED** [review]
Looks good , you can merge.

File: src/pages/homepage/homepage.jsx [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import { Navbar } from "../../component/navbar/navbar";
+import { Footer } from "../../component/footer/footer";
+import { Carousel} from "../../component/carousel/carousel";
+import { ProductBody } from "./product-body";
```
Comment (yashpatel1797) [inline src/pages/homepage/homepage.jsx:4]: create `index.js` file to import components and other files.

**Review by yashpatel1797 — APPROVED** [review]
the file structure looks good.
code is clean.
added one suggestion not mandatory to implement.

File: src/pages/homepage/homepage.jsx (hunk 1 above)
Comment (xii-vi) [reply src/pages/homepage/homepage.jsx:4]: _Thanks will work on suggestion_

**Pull request merged**